style.css

```
@@ -0,0 +1,186 @@
+/* basic grid-layout, sets grid for head, body and footer */
+.grid-layout-body {
+  display: grid;
+  height: 100vh;
+  grid-template-columns: 1fr 280px 1fr; /*1fr 280px 1fr;*/
+  grid-template-rows: auto auto 80px;
+}
+
+.grid-item-header {
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+}
+
+.grid-item-main {
+  grid-column: 2 / 3;
+  grid-row: 2 / 3;
+}
+
+.grid-item-footer {
+  grid-column: 2 / 3;
+  grid-row: 3 / 4;
+}
+/*Basic style for body, Head, main, footer*/
+body {
+  font-family: "Nunito Sans", sans-serif;
+  font-size: 14px;
+  background-color: #e6a937;
+  box-sizing: border-box;
+}
+header {
+  background-color: rgb(166, 213, 219);
+  margin-bottom: 5px;
+  padding: 10px;
+}
+
+main {
+  background-color: deeppink;
+  margin: 5px 0px;
+  padding: 10px;
+}
+
+footer {
+  background-color: darksalmon;
+  margin-top: 5px;
+}
+
+a:visited {
+  color: red;
+}
+a:link {
+  text-decoration: none;
+  color: black;
+}
+/*more detailed styling on different elements from top to bottom*/
+/*header*/
+h1 {
+  text-align: center;
+  font-style: italic;
+  margin-bottom: 0px;
+}
+.flex-nav-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-top: solid 1px;
+}
+.flex-nav-element {
+  font-size: 10px;
+  background-color: darkseagreen;
+  padding: 0px 4px;
+  margin: 2px 0px 0px 0px;
```

```suggestion
  margin: 2px 0 0 0;
```

Actually I think this could be more optimised. But I'm not shure. So maybe this could also work:
```suggestion
  margin: 2px 0 0;
```
Please check!